src/app/body/common-tools/profile-pictures/profile-pictures.component.html

```diff
@@ -0,0 +1,21 @@
+<!--/*********************************************************** 
+* Copyright (C) 2022 
+* Worktez 
+* 
+* This program is free software; you can redistribute it and/or 
+* modify it under the terms of the MIT License 
+* 
+*Author:sanjaykrishna1203@gmail.com
+* 
+* This program is distributed in the hope that it will be useful, 
+* but WITHOUT ANY WARRANTY; without even the implied warranty of 
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+* See the MIT License for more details. 
+***********************************************************/-->
+<ng-container *ngIf="photoUrlReady">
+    <ng-container *ngFor="let item of ProfileList">
+        <span class="avatar">
+            <img [src]='item' onerror= "this.src = '../../assets/defaultavatar.jpg'" width="40" height="40"/>
```

Comment:
Is this path for the default avatar correct?

Reply:
Yes, I took it from the suggestion bucket.

Reply:
The path is wrong there too. Please fix it there as well.